### ISSUE TITLE
rewrite dev example

### ISSUE THREAD

**Issue by icebob** [icebob/vue-form-generator#119]
Now the dev example is too huge and unmanageable. Need to split more small examples.
E.g.
- input fields
- selects
- date pickers
- sliders
- checkbox, switch, radio

- small real example (user settings)
- sandbox (for develop & test new fields)

**Comment by cristijora** [comment]
Is it ok to do a solution with vue router ? Have pages with all the things mentioned above

**Comment by lionel-bijaoui** [comment]
@cristijora yes please, because, if we need to do a different webpack config each time, it is going to be a pain

**Comment by cristijora** [comment]
I can start working on this in a day. Don't know how much time it will take though.

**Comment by icebob** [comment]
Great!

**Comment by cristijora** [comment]
Do you want it very simple and clean or have some bootstrap stuff in it so the examples look more pretty ?

**Comment by icebob** [comment]
I think need bootstrap because need it for some fields. So can be pretty :)

**Comment by lionel-bijaoui** [comment]
Hey @cristijora , what's your progress on this issue ? Any roadblock ? Need help ?

**Comment by cristijora** [comment]
I have some issues with free time. Started 2 pages (inputs and selects) Don't have any roadblock but help would be appreciated. You can see the linked commit from above. Hope I can find some free time by the end of this week but help would be needed indeed here.

**Comment by tomByrer** [comment]
Since you're reworking the examples, should the font conform to the [standard Bootstrap v3 fonts](https://github.com/twbs/bootstrap/blob/v3-dev/less/variables.less#L45-L49)?  Devs tend to copy/paste examples, so they should be as agnostic & flexible as possible IMHO.

**Comment by icebob** [comment]
Any updates about it?

**Comment by dflock** [comment]
I was thinking about having a go at this today, but I'm not going to get time for a while.

I think this should probably be merged with #166. Seems pretty do-able, just finding time.
